test/performance/dataplane-probe/main.go

```diff
@@ -45,7 +45,10 @@ func main() {
 	defer cancel()
 
 	// Use the benchmark key created
-	ctx, q, qclose, err := mako.Setup(ctx)
+	mc, err := mako.Setup(ctx)
+	q := mc.Quickstore
```

Comment:
At this point why not just pass `mc` around?

Reply:
Just trying to make the following function calls look shorter, otherwise for every `q` we need to write as `mc.Quickstore`. Do you want me to change to be like that?

Reply:
How many calls are there, though? :-)

Reply:
In `dataplane-probe` there are about 5, but in `deployment-probe` there are more than 10

Reply:
IDK, either way is fine,
but just `q, ctx, ... := mc.QS, mc.Ctx,...`

Reply:
Done. Thanks